client/app/components/packages/landuse-form/change-city-map.hbs

```diff
@@ -1,367 +1,286 @@
-              data-test-related-acquisition="true"
-            >
-              Yes
-            </RadioButton>
-            <RadioButton
-              @targetValue={{false}}
-              data-test-action-related-acquisition="false"
-            >
-              No
-            </RadioButton>
+          }} />
+    <label for="dcpChangestreetgradeopt">
+      Street grade
+    </label>
+
+    <br>
+
+    <form.Field @attribute="dcpEasement3" @type="checkbox" as |Checkbox|>
+      <Checkbox>
+        Easement
+      </Checkbox>
+    </form.Field>
+
+  </Ui::Question>
+
+  <Ui::Question as |Q|>
+    <Q.Legend>
+      Is there a related acquisition of property?
+    </Q.Legend>
+
+    <form.Field @attribute="dcpRelatedacquisitionofpropertyopt" @type="radio-group" as |RadioGroup|>
+      <RadioGroup @options={{optionset 'landuseForm' 'dcpRelatedacquisitionofpropertyopt' 'list'}} />
+    </form.Field>
+  </Ui::Question>
+
+  <Ui::Question class="no-margin-top" data-test-related-acquisition-radio-group as |Q|>
+    <Q.Legend>
+      Is there a related disposition of property?
+    </Q.Legend>
+
+    <form.Field @attribute="dcpRelatedacquisition" @type="radio" as |RadioButton|>
+      <RadioButton @targetValue={{true}} data-test-related-acquisition="true">
+        Yes
+      </RadioButton>
+      <RadioButton @targetValue={{false}} data-test-action-related-acquisition="false">
+        No
+      </RadioButton>
+    </form.Field>
+
+  </Ui::Question>
+
+  <Ui::Question as |dcpOnlychangetheeliminationofamappedbutunimpQ|>
+    <dcpOnlychangetheeliminationofamappedbutunimpQ.Legend>
+      Is the only change (requested for the City Map) the elimination of a mapped but unimproved street?
+    </dcpOnlychangetheeliminationofamappedbutunimpQ.Legend>
+
+    <form.Field @attribute="dcpOnlychangetheeliminationofamappedbutunimp" @type="radio-group" as
+      |DcpOnlychangetheeliminationofamappedbutunimpRadioGroup|>
+      <DcpOnlychangetheeliminationofamappedbutunimpRadioGroup
+        @options={{optionset 'landuseForm' 'dcpOnlychangetheeliminationofamappedbutunimp' 'list'}} as
+        |dcpOnlychangetheeliminationofamappedbutunimp|>
+        {{#if (eq
+            dcpOnlychangetheeliminationofamappedbutunimp
+            (optionset 'landuseForm' 'dcpOnlychangetheeliminationofamappedbutunimp' 'code' 'Yes')
+          )}}
+        <Ui::Question as |dcpYesmappedbutunimprovedstreetelimatedQ|>
+          <dcpYesmappedbutunimprovedstreetelimatedQ.Label>
+            Is it being eliminated from the property of an owner-occupied, one- or two-family residence?
+          </dcpYesmappedbutunimprovedstreetelimatedQ.Label>
+
+          <form.Field @attribute="dcpYesmappedbutunimprovedstreetelimated" @type="radio-group" as
+            |DcpYesmappedbutunimprovedstreetelimatedRadioGroup|>
+            <DcpYesmappedbutunimprovedstreetelimatedRadioGroup
+              @options={{optionset 'landuseForm' 'dcpOnlychangetheeliminationofamappedbutunimp' 'list'}}>
```

Comment:
This looks like it's trying to use the optionset of the field that's associated with the question above it instead of the `dcpYesmappedbutunimprovedstreetelimated` one. 

Also, I made a note on the optionset.js file about this -- this specific field doesn't have an integer optionset, instead it's just a boolean. In that case we have a "common" reusable optionset for those fields.

https://dcppfsuat2.crm9.dynamics.com/api/data/v9.1/EntityDefinitions(LogicalName=%27dcp_landuse%27)/Attributes(LogicalName=%27dcp_yesmappedbutunimprovedstreetelimated%27)